app/assets/stylesheets/components/icon.scss

```diff
@@ -1,5 +1,27 @@
+@mixin icon-image($url, $size: 1rem) {
+  background: url($url) center no-repeat;
+  background-size: contain;
+  width: $size; height: $size;
+
+  display: inline-block;
+}
+
 @include b(icon) {
-  @include m(left)  { margin-right: vr(.75); }
-  @include m(right) { margin-left: vr(.75);  }
+  @include m(left)  { margin-right: vr(1); }
+  @include m(right) { margin-left: vr(1);  }
   @include m(sub)   { vertical-align: sub;   }
```

Comment:
tiny thing but the spacing looks inconsistent here between the ; and the closing bracket.

Reply:
It is consistent in that it increases by one space each time.... ;) Fixed now.